src/app/components/application/program-info/program-info.component.ts

```diff
@@ -178,6 +183,7 @@ export class ProgramInfoComponent implements OnInit {
             this.modalService.open(modal, {ariaLabelledBy: 'modal-basic-title'}).result
               .then(instrumentResult => {
                 program.selectedInstrument = instrumentResult;
+                program.selectedInstrumentOther = this.otherInstrument;
```

Comment:
Feels weird that one of these is being sent back in the `close` of the modal while the other isn't... but I guess it's fine if it works? @smithmd

Reply:
yea, it's weird, but it's how the private lesson count is handled also, so we mirrored that.